spec/multi_search_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'spec_helper'
+
+describe 'multi-search' do # rubocop:todo RSpec/DescribeClass
+  def reset_indexes
+    [Book, Color, Product].each do |klass|
+      klass.delete_all
+      klass.index.delete_all_documents
+    end
+  end
+
+  before(:all) { reset_indexes } # rubocop:todo RSpec/BeforeAfterAll
+
+  after { reset_indexes }
```

Comment:
Why not just a regular before each?

Reply:
If the indexes are not reset in an `after`, they would pollute tests in `integration_spec` (of course if this test happens to be run before them).

Unfortunately rubocop was being really annoying about `before(:all)` and `after(:all)` even though they are perfectly valid in this situation.

Reply:
I mean, why not just have a simple: `before { reset_indexes }` that should run for each spec? Instead of having the after and a before(:all)?

Reply:
I got a failure on the suite without at least an `after(:all)` since the documents seemed to be leaking into other tests, but now that I try to recreate it manually it does not seem to want to happen in any way.

I'll replace it with a before.